CHANGELOG.md

```diff
@@ -8,14 +8,37 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
-### Added
+## Added
 
 - ERC1155 component and preset (#896)
+- Mixin implementations in components (#863)
+  - `AccessControlComponent::AccessControlMixinImpl`
+  - `OwnableComponent::OwnableMixinImpl`
+  - `OwnableComponent::OwnableTwoStepMixinImpl`
+  - `AccountComponent::AccountMixinImpl`
+  - `EthAccountComponent::EthAccountMixinImpl`
+  - `ERC20Component::ERC20MixinImpl`
+  - `ERC721Component::ERC721MixinImpl`
+  - `ERC721Receiver::ERC721ReceiverMixinImpl`
+- ERC721Component functions and Storage member
+  - `InternalTrait::_set_base_uri` and `InternalTrait::_base_uri` to handle ByteArrays (#857)
+  - `ERC721_base_uri` Storage member to store the base URI (#857)
 
 ### Changed
 
 - Change unwrap to unwrap_syscall (#901)
-- Integrate ByteArray into ERC20 and ERC721 (#857)
+- ERC20Component
+  - `IERC20::name` and `IERC20::symbol` return ByteArrays instead of felts (#857)
+- ERC721Component
+  - `IERC721::name`, `IERC721::symbol`, and `IERC721Metadata::token_uri` return ByteArrays instead of felts (#857)
+  - `InternalTrait::initializer` accepts an additional `base_uri` ByteArray parameter (#857)
+  - IERC721Metadata SRC5 interface ID. This is changed because of the ByteArray integration (#857)
+
+### Removed
+
+- ERC721Component function and Storage member
+  - `InternalTrait::_set_token_uri` because full token URIs are no longer stored (#857)
+  - `ERC721_token_uri` Storage member because full token URIs are longer stored (#857)
```

Comment:
```suggestion
  - `InternalTrait::_set_token_uri` because individual token URIs are no longer stored (#857)
  - `ERC721_token_uri` Storage member because individual token URIs are longer stored (#857)
```
maybe this is more clear? not sure